run_test.py

```diff
@@ -48,25 +47,96 @@ def make_request(self):
         PermanentRequest.csrf = json_resp["csrf"]
 
 
+def get_presigned_url(record_vo):
+    body = [{
+        'RecordVO': record_vo,
+        'SimpleVO': {
+            'key': 'type',
+            'value': 'application/octet-stream',
+        },
+    }]
+    request = PermanentRequest('/api/record/getPresignedUrl', data=body)
+    return request.response['SimpleVO']['value']
+
+
+def upload_to_s3(s3_url, file_path, presigned_post):
+    fields = presigned_post['fields']
+    with open(file_path, 'rb') as f:
+        fields['Content-Type'] = 'application/octet-stream'
+        logging.debug('Data to be form encoded for S3:')
```

Comment:
Do we need these logging statements?

Reply:
I think you're right that we could improve the logging statements!

I think there are some useful, debug-level logging statements that we should include, as these did help us during development, and I'll think a little more about what those should be.

Reply:
Added a new commit that cleans up the logging a bit. Thanks for the flag, @mithuna!